src/options.cpp

```diff
@@ -1157,6 +1157,18 @@ void options_manager::init()
         true, COPT_CURSES_HIDE
         );
 
+    add( "PIXEL_MINIMAP_MODE", "graphics", translate_marker( "Pixel Minimap drawing mode" ),
+        translate_marker( "Specified the mode in which the minimap drawn." ), {
+            { "solid", translate_marker( "Solid" ) },
+            { "squares", translate_marker( "Squares" ) },
+            { "dots", translate_marker( "Dots" ) } }, "dots", COPT_CURSES_HIDE
```

Comment:
Since the default tileset ist ChestHole the default drawing mode should probably be solid.

Reply:
I guess you're right. Dots are, on the contrary, too dark for ChestHole. Maybe Squares could be the default choice instead? It's not that I try to push the feature forward, I simply like their appearance more.